bundles/org.openhab.binding.dirigera/src/main/resources/OH-INF/thing/color-light.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dirigera"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="color-light">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Color Light</label>
+		<description>Light with color support</description>
+
+		<channels>
+			<channel id="power" typeId="system.power">
+				<label>Light Powered</label>
+				<description>Power state of light</description>
+			</channel>
+			<channel id="brightness" typeId="system.brightness">
+				<label>Brightness</label>
+				<description>Brightness of light in percent</description>
+			</channel>
+			<channel id="color-temperature" typeId="system.color-temperature">
```

Comment:
As mentioned above, consider to add a `system.color-temperature-abs` channel as well; and provide the state description unit, min/max kelvin

Reply:
Done